### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by Cerdtruir**
Added the list structure generated with js. using the 1_list_structure.md spec
To view this you need to use webpack

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,19 @@
...
+
+## Built With
+
+- Webpack
+- HTML
+- CSS
+- JavaScript
+
+## Live Demo
+
```
Comment (ikraamg) [inline README.md:13]: Screenshot and links when this is merged 👌

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+const todoItems = [
+  { index: 1, description: 'Eat Lunch', completed: false },
+  { index: 0, description: 'Eat Breakfast', completed: false },
+  { index: 2, description: 'Eat Supper', completed: false },
+];
+
+const tasksHTML = function () {
+  let currentIndex = 0;
+  for (let i = 0; i < todoItems.length; i += 1) {
```
Comment (ikraamg) [inline src/index.js:13]: Could use `forEach` instead:
```js
users.forEach((user, index)=>{
	console.log(index); // Prints the index at which the loop is currently at
});
```

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    const task = todoItems[i];
+    if (task.index === currentIndex) {
+      const content = document.createElement('li');
+      content.innerHTML = `<span class="checkbox">&#9634;</span> ${task.description} <span class="list-span list-move">&#xFE19;</span>`;
```
Comment (ikraamg) [inline src/index.js:17]: This innerHTML is dangerous as it takes user input.

**Review by ikraamg — APPROVED** [review]
Looks good 👍

**Comment by Cerdtruir** [comment]
I'm closing this pr, I made  a new one with the latest branch. I moved your comments to that one

**Comment by Cerdtruir** [comment]
I'm closing this pr, I made a new one with the latest branch. I moved your comments to that one

**Pull request closed without merging**